recipes-kernel/linux/files/patches-5.10/0050-arm64-dts-ti-iot2050-Add-layout-of-OSPI-flash.patch

```diff
@@ -0,0 +1,65 @@
+From 7da3b3dec3595337ec039eb5e8a5c0bf6faef8f3 Mon Sep 17 00:00:00 2001
+From: Jan Kiszka <jan.kiszka@siemens.com>
+Date: Mon, 21 Mar 2022 15:53:15 +0100
+Subject: [PATCH 050/119] arm64: dts: ti: iot2050: Add layout of OSPI flash
+
+Describe the layout of the OSPI flash as the latest firmware uses it.
+Specifically the location of the U-Boot envs is important for userspace
+in order to access it.
+
+Signed-off-by: Jan Kiszka <jan.kiszka@siemens.com>
+---
+ .../boot/dts/ti/k3-am65-iot2050-common.dtsi   | 37 ++++++++++++++++++-
+ 1 file changed, 35 insertions(+), 2 deletions(-)
+
+diff --git a/arch/arm64/boot/dts/ti/k3-am65-iot2050-common.dtsi b/arch/arm64/boot/dts/ti/k3-am65-iot2050-common.dtsi
+index 65da226847f4..f3df67e5d712 100644
+--- a/arch/arm64/boot/dts/ti/k3-am65-iot2050-common.dtsi
++++ b/arch/arm64/boot/dts/ti/k3-am65-iot2050-common.dtsi
+@@ -603,8 +603,41 @@ flash@0 {
+ 		cdns,tchsh-ns = <60>;
+ 		cdns,tslch-ns = <60>;
+ 		cdns,read-delay = <2>;
+-		#address-cells = <1>;
+-		#size-cells = <1>;
++
++		partitions {
++			compatible = "fixed-partitions";
++			#address-cells = <1>;
++			#size-cells = <1>;
++
++			seboot@0 {
++				label = "seboot";
++				reg = <0x0 0x180000>;
++			};
++			tispl@180000 {
++				label = "tispl";
++				reg = <0x180000 0x200000>;
++			};
++			u-boot@380000 {
```

Comment:
comments can be added to describe the size of the partition for future maintenance
like the Notes wic 64K 2M and so on.

Reply:
Let me try...